packages/node_modules/@node-red/nodes/locales/de/messages.json

```diff
@@ -596,11 +596,11 @@
       "regex" : "Reguläre Ausdrücke verwenden"
     },
     "action" : {
-      "set" : "Festlegen",
+      "set" : "Setze",
```

Comment:
Maybe change this line to "Setzen" to make it consistent with the other three options of the change node.

Reply:
@ketzu Are you still going to make this change?